Cargo.toml

```diff
@@ -40,6 +40,11 @@ walker = "^1.0.0"
 [target.x86_64-unknown-linux-gnu.dependencies.inotify]
 version = "^0.1"
 
+[target.x86_64-unknown-linux-gnu.dependencies.mio]
+git = "https://github.com/carllerche/mio.git"
```

Comment:
It is my understanding that libraries which depend on libraries _not in Crates.io_ cannot be published to Crates.io. Has this changed? If not, I cannot accept this PR until it points to a Crates.io-released version of `mio`.